### PR TITLE
android: Prompt for install updates when they're downloaded

### PR DESCRIPTION
**Pull request by pazos**
Requires https://github.com/koreader/android-luajit-launcher/pull/303 and https://github.com/koreader/koreader-base/pull/1367.

When the new APK is downloaded and the app resumed a confirmbox will ask if you want to update. The android notification of the download is now disabled.

Since the confirmbox could clash with other resume methods it will take priority over them.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7632)

### CODE REVIEW

File: frontend/device/android/device.lua [hunk 1]
```diff
@@ -445,6 +453,19 @@ function Device:untar(archive, extract_to)
     return android.untar(archive, extract_to)
 end
 
+function Device:install()
+    local UIManager = require("ui/uimanager")
+    local ConfirmBox = require("ui/widget/confirmbox")
+    UIManager:show(ConfirmBox:new{
+        text = _("Update ready, do you want to install?"),
```
Comment (NiLuJe) [inline frontend/device/android/device.lua:460]:
```suggestion
        text = _("Update ready, do you want to install it now?"),
```

Comment (Frenzie) [reply frontend/device/android/device.lua:460]: Is there a reason not to want to use our existing (already translated etc.) text?

Comment (pazos) [reply frontend/device/android/device.lua:460]: The one in https://github.com/koreader/koreader/blob/master/frontend/ui/otamanager.lua#L54 is incorrect in android.

But maybe we can hide platform details an use the same prompt everywhere.

Comment (Frenzie) [reply frontend/device/android/device.lua:460]: Oh right, I forgot the exact phrasing.

Indeed, we should probably go for something more generic like:

```
Update is ready. Install it now?

Cancel              Install
```

**Review by Frenzie — COMMENTED** [review]
lgtm besides the one luacheck issue about Event now being unused.

**Pull request merged**